packages/fxa-auth-server/bin/key_server.js

```diff
@@ -71,10 +71,14 @@ async function run(config) {
     Container.set(AuthFirestore, authFirestore);
   }
 
-  const redis = require('../lib/redis')(
+  const sessionTokensRedis = require('../lib/redis')(
     { ...config.redis, ...config.redis.sessionTokens },
     log
   );
+  const authServerCacheRedis = require('../lib/redis')(
+    { ...config.redis, ...config.redis.authServerCache },
+    log
+  );
```

Comment:
nit: since we're doing the `require('../lib/redis')` twice now (here and L74) should we move it up to a more global `const redis = require('redis');` on L43.5 and convert these to:
```js
  const sessionTokensRedis = redis(
    { ...config.redis, ...config.redis.sessionTokens },
    log
  );
  const authServerCacheRedis = redis(
    { ...config.redis, ...config.redis.authServerCache },
    log
  );
```